tests/functional/steps/given.py

```diff
@@ -67,6 +68,16 @@ def user_directory() -> UserDirectory:
     return UserDirectory()
 
 
+@given(
+    parsers.cfparse(
+        "I am a {user_name:String} user with the NHS number linked to a {user_name:String} account",
```

Comment:
Do we need to reference user_name twice? Can it only be referenced once in the string?
E.g "I am a {user_name:String} user with the NHS number linked to an account"

Reply:
Also, might want to rename that parameter to something like user_level?